packages/discovery-react-components/src/components/CIDocument/components/CIDocument/CIDocument.tsx

```diff
@@ -43,10 +43,10 @@ import {
   Field,
   Attributes,
   Relations
-} from 'components/CIDocument/types';
+} from '../../types';
 import { Address, Mention } from '../MetadataPane/types';
 import { Items } from '../DetailsPane/types';
-import { defaultTheme, Theme } from 'utils/theme';
+import { defaultTheme, Theme } from '../../../../utils/theme';
```

Comment:
Are these regressions? Didn't we add code to avoid having to do these relative paths?

Reply:
I think `january_fix_pack` branch did not have the fix which is there in develop which prevents the relative path, that's why the relative paths are there in fixpack branch, and now it is coming on top of `develop` again.